### DISCUSSION TITLE
Dont know when celery_taskmeta and celery_tasksetmeta are being created

### DISCUSSION THREAD

**Discussion by marioabz** [celery/celery#8818] [Q&A] [answered]
I configured a flask project with Postgres as db using flask-sqlalchemy and then also use as backend on Celery the same database and schema.
I created a model with a foreign ket to celery_tasks and modified the metadata of our db so that it realizes that. we have celery_taskmeta. So far it works but then, when testing on a pipeline when I run migrations (alembic) it does no detect the table celery_taskmeta.

Is there a way to explicitly create Celery task models on a schema or can you point me to the right documentation section?
Thanks in advance!

**Comment by mrxsal** [comment, marked as answer]
> When SQLAlchemy is configured as the result backend, Celery automatically creates two tables to store result meta-data for tasks. This setting allows you to customize the schema of the tables:

https://docs.celeryq.dev/en/stable/userguide/configuration.html#database-table-schemas
